webapp/src/main/webapp/js/officeonline.js

```diff
@@ -288,6 +288,7 @@
     // Constants:
     var DOCUMENT_SAVED = "DOCUMENT_SAVED";
     var FILE_RENAME = "File_Rename";
+    var FILE_VERSIONS = "UI_FileVersions";
```

Comment:
We can use ES6 `const` to mark our constants be really ones. Platform 6 requires ES6 compliant browser only.

Reply:
Fixed: https://github.com/exo-addons/office-online/pull/28/commits/09dd43c4c6497a4cd87cc727a522b7c45c3eefdd